### DISCUSSION TITLE
使用过程中几个小建议

### DISCUSSION THREAD

**Discussion by Meeeeet6** [AlistGo/alist#1136] [Ideas]
使用作者大大的alist有段时间了，有些许小建议，想给作者大大提一下:
1.对盘内文件进行检测 若检测到password.txt则读取内容并对该目录进行加密，感觉有时这比在元信息那设置加密要方便些许~
2.在后台账号设置页内，加多一栏备注栏（进在后台可见，对外不可见），有时分不清账号，代码不对这栏进行引用，仅作个人添加账号时备注所用。

两个小意见希望作者大大有空可以考虑下，谢谢作者大大啦，辛苦啦~

**Comment by anwen-anyi** [comment]
1.之前的V1就是这样做的,不过现在元信息方式也挺简单吧（个人感觉还是现在这种元信息方式加密码方便 毕竟可以加N个不同目录的密码,要是按照之前的那种 你得给云盘上传文件才行 不方便感觉~）
2.QAQ第二点儿确实添加一个备注信息栏 哈哈~ 不过我现在的方式是用**虚拟路径来区分的**不同网盘取不同的名字,**比如下面**


- 国内
   - /国内/AliDriver
   - /国内/189Cloud
   - /国内/189CloudFamily
   - /国内/139Cloud
   - /国内/139CloudFamily
   - /国内/Baidu
   - /国内/Baidu-1
   - /国内/Baidu-2
   - /国内/xxxxx网盘
 - 国外
   - /国外/OneDriver
   -  /国外/其他网盘
 - 其他
   - 其他/腾讯COS
   - 其他/阿里OSS
   - 其他/天翼OOS
   - 其他/七牛Kodo 

以上仅属于个人看法~~~不喜勿喷 哈哈QAQ 友好交流下~

**Reply by Meeeeet6** [reply]
没有没有，都是自己使用过程中的一些小建议嘛，都挺好的~
  只是我个人觉得元信息加密像我这种小目录比较多的，感觉有点麻烦，如果复制password.txt文件就快很多，才提出这个建议的，要是可以两全其美就更好啦~
  至于第二条建议，备注信息栏我的用意主要用来区分网盘的，是打算用来备注一些关于网盘的信息例如账号(有些网盘不显示账号，只显示cookie等)啊，一些个人信息啊，又不想对外公开可见这样的。

都挺好的啦，我觉得都可以考虑下，要看作者大大~

**Reply by anwen-anyi** [reply]
> 没有没有，都是自己使用过程中的一些小建议嘛，都挺好的~
> 
>   只是我个人觉得元信息加密像我这种小目录比较多的，感觉有点麻烦，如果复制password.txt文件就快很多，才提出这个建议的，要是可以两全其美就更好啦~
> 
>   至于第二条建议，备注信息栏我的用意主要用来区分网盘的，是打算用来备注一些关于网盘的信息例如账号(有些网盘不显示账号，只显示cookie等)啊，一些个人信息啊，又不想对外公开可见这样的。
> 
> 
> 
> 都挺好的啦，我觉得都可以考虑下，要看作者大大~

嘿嘿  我晓得你什么意思
1.这个你要是要是密码改密码还得去网盘修改文件 上传一个文件  这个确实不如元信息方便管理
2.我的意思是现阶段没有更好的办法来区分，可以用我提到的那种来区分一下先，等后面如果上线了在修改一下，还有就是如果你不想公开的文件夹比较多的话你可以去元信息设置一下 只显示那些…哈哈这样比不显示那些好加
比如你有20个网盘添加了  隐藏其中的18个 以前你得写18个  现在新版的只写两个你想显示的就好 哈哈
